engine-comparison/dispatcher.sh

```diff
@@ -5,61 +5,96 @@
 . $(dirname $0)/../common.sh
 . ${SCRIPT_DIR}/common-harness.sh
 
+
 build_engine() {
-  # [[ ! -e ~/$FENGINE_CONFIGS_DIR/$1 ]] && echo "cant do" && exit 1
-  # All of this code will probably be in a separate script
 
   FENGINE_CONFIG=$1
-
+  [[ ! -e $FENGINE_CONFIG ]] && echo "File DNE" && exit 1
   echo "Building $FENGINE_CONFIG"
   FENGINE_NAME=$(basename $FENGINE_CONFIG)
-  ENGINE_DIR=$WORK/fuzz-engines/${FENGINE_NAME}
-  rm -rf $ENGINE_DIR
-  mkdir $ENGINE_DIR
-
-  # Fuzzing activities go here
+  FENGINE_DIR=$WORK/fengine-builds/${FENGINE_NAME}
+  rm -rf $FENGINE_DIR
+  mkdir $FENGINE_DIR
+
+  . $FENGINE_CONFIG
+  # Build either engine
+
+  if [[ $FUZZING_ENGINE == "libfuzzer" ]]; then
+    echo "Making a version of libfuzzer"
+    svn co http://llvm.org/svn/llvm-project/llvm/trunk/lib/Fuzzer $FENGINE_DIR
+    export LIBFUZZER_SRC=$FENGINE_DIR
```

Comment:
LIBFUZZER_SRC is used in each of the `build.sh` scripts for building benchmarks, so I change this to build the benchmark with our particular fengine rather than the default fengine in `common.sh`. Also, that default fengine won't exist, unless the user provides a corresponding config file.